rpc/glfs-operations.c

```diff
@@ -307,8 +350,10 @@ glusterBlockResizeEntry(struct glfs *glfs, blockModifySize *blk,
 
  out:
   if (ret) {
-    GB_ASPRINTF (errMsg, "Not able to resize storage for %s/%s [%s]",
-                 blk->volume, blk->block_name, strerror(*errCode));
+    if (errMsg && !(*errMsg)) {
```

Comment:
Is this sneaking in an independent change that should be a different patch/fix?
Or else please explain how this is related to the 10MiB space reservation change.

Reply:
Oh yes! In absence of this, the errMsg string that we filled at glusterBlockCheckAvailableSpace() see line 311, will be overridden.

Reply:
Oh, that treatment of errMsg is just plain horrible! :-)
But I get it. That complaint is not for this patch then ... ;-)